src/ralph/back_office/models.py

```diff
@@ -214,11 +215,32 @@ class BackOfficeAsset(Regionalizable, Asset):
         default=None,
         verbose_name=_('Last status change')
     )
+    service_cost = models.IntegerField(
+        default=None,
+        null=True,
+        blank=True,
+        verbose_name=_('Service Cost')
+    )
 
     class Meta:
         verbose_name = _('Back Office Asset')
         verbose_name_plural = _('Back Office Assets')
 
+    @property
+    def service_calculation(self):
+        if self.service_cost is None:
+            return None
+        else:
+            today = date.today()
+            timedelta = (self.buyout_date - today)
+            months = timedelta.days // 30
+            service = (int(self.service_cost) / months)
+            buy_rate = (self.price.amount / self.depreciation_rate) + 50
```

Comment:
why/what is 50?

Reply:
this is pattern what I receive to calculate profitability.

We can discuss this hole pattern on PM, if needed.